packages/caliper-fabric/test/utils/GenerateConfiguration.js

```diff
@@ -64,5 +79,23 @@ class GenerateTestConfiguration {
     _deepCloneObject(object) {
         return JSON.parse(JSON.stringify(object));
     }
+
+    /**
+     *
+     * @param {*} object b
+     * @param {*} propertyName b
```

Comment:
currently the caliper code base has full descriptions in the JSDoc, would be nice to be consistent here even though the JSDoc is not published